src/modules/items.haml

```diff
@@ -367,36 +406,78 @@
                     <br/>
                     ### [Enchantments](id:enchantments)
 
-                    Items and armor can have custom enchantments by using the `enchantment` attribute. Enchantments are specified by their [bukkit name](/reference/enchantments) and the enchantment level separated by a colon `:`. Multiple enchantments are separated by a semicolon `;`.
+                    Any enchantment can be applied to any item. The enchantment type can be specified by
+                    [Minecraft name](http://minecraft.gamepedia.com/Data_values#Enchantment_IDs) or
+                    [Bukkit name](/reference/enchantments).
```

Comment:
IMO, this sentence doesn't sound grammatical correct.
Maybe use "Minecraft enchantment names" & "Bukkit enchantment names" to match the sentence for potion effects, or, "can be specified by their".